File: Assignment 4 copy/Employee.java
public class Employee extends Person {

	private String department;
	
Employee(String newName, String address, String phone, String department) {
	
	super(newName,address,phone);
	this.department = department;
}
	

void setDepartment(String department) {
	
	this.department = department;
	
	//this sets it as belonging to person and not parameter
}

String getDepartment() {
	
	return(this.department);
	
	
}
public String toString() {
	return super.toString() +  "\nDepartment: " + department;
	
	
}
	
}

File: Assignment 3 backup copy/Employee.py
from Person import Person
class Employee(Person):
    def __init__(self,newName = "none",address = "none",phone = "999-999-9999",department = "not assigned"):
         super().__init__(newName,address,phone)
         self.department = department
    
    
    def getDepartment(self):
         return self.department
    
    def setDepartment(self,department):
         self.department = department

    def __str__(self) -> str:
         return super().__str__() + "Department" + self.department
    
    